### PR TITLE
zh-cn: update "Media buffering, seeking, and time ranges"

### PR DESCRIPTION
**Pull request by yin1999**
### Description

update "Media buffering, seeking, and time ranges"

### CODE REVIEW

File: files/zh-cn/web/media/audio_and_video_delivery/buffering_seeking_time_ranges/index.md [hunk 1]
```diff
@@ -1,41 +1,42 @@
 ---
-title: Media buffering, seeking, and time ranges
+title: 媒体缓冲、拖动和时间范围
 slug: Web/Media/Audio_and_video_delivery/buffering_seeking_time_ranges
+l10n:
+  sourceCommit: cfe622c944a58809c185aa71f6d5885f83cdfc3a
```
Comment (yin1999) [inline files/zh-cn/web/media/audio_and_video_delivery/buffering_seeking_time_ranges/index.md:5]:
```suggestion
  sourceCommit: 5aa89c0c0dd2aded106b8d6674fe5370e270f5e6
```

File: files/zh-cn/web/media/audio_and_video_delivery/buffering_seeking_time_ranges/index.md [hunk 2]
```diff
@@ -1,41 +1,42 @@
...
 
-- `length`: The number of time ranges in the object.
-- `start(index)`: The start time, in seconds, of a time range.
-- `end(index)`: The end time, in seconds, of a time range.
+- `length`：对象包含的时间范围的数量。
+- `start(index)`：时间范围的开始时间（以秒为单位）。
+- `end(index)`：时间范围的结束时间（以秒为单位）。
 
-没有用户操作的话通常只有一个时间范围，但是如果你在媒体中跳来跳去那么就会出现多个时间范围，下面形象化的表名了这一点。This represents two buffered time ranges — one spanning 0 to 5 seconds and the second spanning 15 to 19 seconds.
+没有用户交互的情况下通常只有一个时间范围，但是如果你在媒体中跳来跳去，那么就会出现多个时间范围，下面形象地表名了这一点。这表示两个缓冲的时间范围——第一个范围从 0 到 5 秒，第二个范围从 15 到 19 秒。
```
Comment (jasonren0403) [inline files/zh-cn/web/media/audio_and_video_delivery/buffering_seeking_time_ranges/index.md:39]:
```suggestion
没有用户交互的情况下通常只有一个时间范围，但是如果你在媒体中跳来跳去，那么就会出现多个时间范围，下面的示意图形象地表明了这一点。这表示两个缓冲的时间范围——第一个范围从 0 到 5 秒，第二个范围从 15 到 19 秒。
```

File: files/zh-cn/web/media/audio_and_video_delivery/buffering_seeking_time_ranges/index.md [hunk 3]
```diff
@@ -67,68 +68,68 @@ myAudio.buffered.end(1); // returns 19
...
-![A simple audio player with play button, seek bar and volume control, with a series of red rectangles beneath it representing time ranges.](bufferedtimeranges.png)
+![一个简单的音频播放器，带有播放按钮、搜索栏和音量控制，下面有一系列代表时间范围的红色矩形。](bufferedtimeranges.png)
 
-> **备注：** You can see the [timerange code running live on JS Bin](http://jsbin.com/memazaro/1/edit).
+> **备注：** 你可以查看 [JS Bin 上实时运行的时间范围代码](https://jsbin.com/memazaro/1/edit)。
 
 ## Seekable
 
-`Seekable` 属性返回一个 {{ domxref("TimeRanges") }} 对象告诉我们哪一部分媒体可以不等待的播放; this is irrespective of whether that part has been downloaded or not. Some parts of the media may be seekable but not buffered if byte-range requests are enabled on the server. Byte range requests allow parts of the media file to be delivered from the server and so can be ready to play almost immediately — thus they are seekable.
+`seekable` 属性返回一个 {{domxref("TimeRanges")}} 对象，告诉我们媒体的哪一部分可以不经过延迟直接播放；这与该部分是否已被下载无关。如果服务器上启用了字节范围请求，媒体的一些部分可能是可查找（seekable）但又没有缓冲的，字节范围请求允许服务器分发媒体文件的某一部分，因此可以几乎立即准备好播放——因此它们是可查找的。有关字节范围请求的更多信息，请参见 [HTTP 范围请求](/zh-CN/docs/Web/HTTP/Range_requests)。
```
Comment (jasonren0403) [inline files/zh-cn/web/media/audio_and_video_delivery/buffering_seeking_time_ranges/index.md:110]: 两个连着的“因此”读上去有些怪

```suggestion
`seekable` 属性返回一个 {{domxref("TimeRanges")}} 对象，告诉我们媒体的哪一部分可以不经过延迟直接播放；这与该部分是否已被下载无关。如果服务器上启用了字节范围请求，媒体的一些部分可能是可查找（seekable）但又没有缓冲的，字节范围请求允许服务器分发媒体文件的某一部分，因此可以几乎立即准备好播放——故它们是可查找的。有关字节范围请求的更多信息，请参见 [HTTP 范围请求](/zh-CN/docs/Web/HTTP/Range_requests)。
```

**Review by jasonren0403 — APPROVED** [review]
LGTM, just two improvments

**Pull request merged**